code/modules/antagonists/disease/disease_event.dm

```diff
@@ -3,8 +3,7 @@
 	name = "Spawn Sentient Disease"
 	typepath = /datum/round_event/ghost_role/sentient_disease
 	weight = 7
-	//max_occurrences = 1 //ORIGINAL
-	max_occurrences = 0 //SKYRAT EDIT CHANGE
+	max_occurrences = 1
```

Comment:
Not related to the ambitions system, at the _least_ make sure its change-logged if not atmoized